### DISCUSSION TITLE
Would it be possible to use something like this script for sports live streams?

### DISCUSSION THREAD

**Discussion by ttx99** [justchokingaround/lobster#29] [Q&A]
https://streamseast.ai/
https://1stream.eu/

**Comment by justchokingaround** [comment]
yes it is. i'll write a short script for it in the upcoming days (if i dont forget)

**Comment by justchokingaround** [comment]
i can write a short scraper for either of these websites but they seem to be very unstable

**Comment by justchokingaround** [comment]
no, you can't do that bc u'll just get a 404 on the stream. even if u want to have auto-reconnect, u'll have to wait for minutes

**Comment by justchokingaround** [comment]
i'll write 2 scripts for both when i have some free time

**Reply by justchokingaround** [reply]
will check it out in a month, if u remind me (sorry)

**Comment by ttx99** [comment]
https://streamseast.ai/
https://1stream.eu/
![1](https://github.com/justchokingaround/lobster/assets/95531944/f16adcbb-6a92-48cd-adb4-2cbd32346ab0)
![3](https://github.com/justchokingaround/lobster/assets/95531944/2fab3a9d-7955-44c9-8c27-0b9375ff7f58)
